src/handlebar-helpers.js

```diff
@@ -29,6 +29,14 @@ const helpers = {
       return data.split(delimiter);
     }
     return [data];
+  },
+  divide: function (v1, v2) {
+    if ( v2 != 0 ) {
```

Comment:
lets validate that the input is a number

```suggestion
    if ( typeof x === 'number' && typeof y === 'number' && y !== 0) {
```